README.md

```diff
@@ -37,6 +40,14 @@ cd dspot
 mvn clean install -DskipTests
 ```
 
+### Install Diff-JJoules-Maven
+
+```sh
+git clone //url
```

Comment:
Since Diff-JJoules-maven is distributed with Diff-JJoules, there is no need to clone.